# Algortihm Parameters
LR_A: 0.001
LR_C: 0.001
GAMMA:  0.9
TAU:  0.01
MEMORY_CAPACITY:  300000
BATCH_SIZE: 32
MAX_EPSIODES: 900
MAX_STEPS:  300

# Environment Parameters
desired_pose:
    x: 2.0
    y: 0.0
    z: 0.6
desired_force: 7.08 # In Newtons, normal contact force when standing still with 9.81 gravity
desired_yaw: 0.0 # Desired yaw in radians for the hopper to stay
max_height: 10.0   # in meters
min_height: 0.1   # in meters
max_incl: 1.57       # in rads
running_step: 0.001   # in seconds
joint_increment_value: 0.05  # in radians
done_reward: -1000 # reward
alive_reward: 100 # reward
fail_reward: -1000.0

weight_r1: 1.0 # Weight for joint positions ( joints in the zero is perfect )
weight_r2: 0.0 # Weight for joint efforts ( no efforts is perfect )
weight_r3: 1.0 # Weight for contact force similar to desired ( weight of monoped )
weight_r4: 1.0 # Weight for orientation ( vertical is perfect )
weight_r5: 1.0 # Weight for distance from desired point ( on the point is perfect )
